sys-boot/grub/grub-2.06-r2.ebuild

```diff
@@ -46,78 +88,73 @@ REQUIRED_USE="
 	grub_platforms_loongson? ( fonts )
 "
 
-# os-prober: Used on runtime to detect other OSes
-# xorriso (dev-libs/libisoburn): Used on runtime for mkrescue
-RDEPEND="
-	app-arch/xz-utils
-	>=sys-libs/ncurses-5.2-r5
-	debug? (
-		sdl? ( media-libs/libsdl )
-	)
-	device-mapper? ( >=sys-fs/lvm2-2.02.45 )
-	libzfs? ( sys-fs/zfs )
-	mount? ( sys-fs/fuse )
-	truetype? ( media-libs/freetype:2= )
-	ppc? ( sys-apps/ibm-powerpc-utils sys-apps/powerpc-utils )
-	ppc64? ( sys-apps/ibm-powerpc-utils sys-apps/powerpc-utils )
-"
-DEPEND="${RDEPEND}
+BDEPEND="
 	${PYTHON_DEPS}
-	app-misc/pax-utils
 	sys-devel/flex
 	sys-devel/bison
 	sys-apps/help2man
 	sys-apps/texinfo
 	grub_platforms_arm64? ( cross-aarch64-cros-linux-gnu/gcc )
```

Comment:
Here you dropped the `fonts ( freetype, pkgconfig }`. I think it's unnecessary difference between our ebuild and Gentoo - we don't enable fonts USE flag anyway, so it could stay as is.